src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -7,23 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <WebpackInputs Include="src\**\*.ts" />
-    <WebPackOutputs Include="dist\Debug\blazor.webassembly.js" />
-    <WebPackOutputs Include="dist\Release\blazor.webassembly.js" />
-    <WebPackOutputs Include="dist\Debug\blazor.server.js" />
-    <WebPackOutputs Include="dist\Release\blazor.server.js" />
+    <ProjectReference Include="..\..\SignalR\clients\ts\signalr-protocol-msgpack\signalr-protocol-msgpack.npmproj" />
   </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
 
-  <!-- Override the default 'Build' target from eng/targets/Npm.Common.targets. -->
-  <Target Name="Build"
-          Inputs="@(WebpackInputs)"
-          Outputs="@(WebPackOutputs)">
-    <RemoveDir Directories="dist" />
-
-    <Yarn Command="run build:debug" />
-    <Yarn Command="run build:production" />
-  </Target>
+  <Target Name="Restore" />
```

Comment:
I'm sure this is all for a good reason but could you explain what it is?

Previously the `WebpackInputs`/`WebPackOutputs` were for incremental build. Is that handled in another way now, or are we losing that feature?

Is the `ProjectReference` to `signalr-protocol-msgpack.npmproj` just a way of ensuring the SignalR library builds first, or are we changing to a source reference instead of an NPM reference now?

Reply:
Sorry, did not see your comments until after I merged. There's a couple of issues here: since we're compiling SignalR source files, we have to also include that closure to determine incrementalism. However these project files currently don't have a way to specify that. Secondly, restoring a `file://` results in the directory being copied to the `node_modules`. Arcade discovers these directories and attempts to build them, which works out poorly since none of the other files \ folders required to build these project files exist. 

Building this is much slower than earlier, is that something we want to try track and address?

Reply:
> Building this is much slower than earlier, is that something we want to try track and address?

Yes. Obviously not for preview 6, but this is worth following up on.

We're taking on a lot of technical debt around NPM. The amount of nonstandardness in our build system, and its tendency to always increase over time, is problematic.

As soon as preview 6 ships and hence SignalR's NPM package is on the public npmjs.org, can we go back to having a regular package dependency on the published package? These two areas of the product should be pretty decoupled and it would help if we could eliminate the `package.json`-rewriting and source-to-source nonstandardness.